iOS Client/PodcastsController/MBPodcastsViewController.swift

```diff
@@ -63,10 +67,18 @@ class MBPodcastsViewController: UIViewController, UITableViewDataSource, UITable
         // swiftlint:enable force_cast
         if indexPath.row < podcasts.count {
             let podcast = podcasts[indexPath.row]
-            cell.configure(title: podcast.title ?? "", image: UIImage(named: podcast.image), date: podcastDateFormatter.string(from: podcast.pubDate))
+            let saved = savedPodcastTitles?.contains(podcast.title ?? "") ?? false
```

Comment:
let's make title required in the domain and reject DTOs without a title just to be safe